plugins/globalias/globalias.plugin.zsh

```diff
@@ -0,0 +1,11 @@
+globalias() {
+   zle _expand_alias
+   zle expand-word
+   zle self-insert
+}
+zle -N globalias
+bindkey -e " " globalias
+bindkey -v " " globalias
+bindkey -e "^ " magic-space           # control-space to bypass completion
```

Comment:
@mcornella Thanks for this review and for showing me ```-M``` key

I switched to strict 'emacs' and 'viins' modes and so 'isearch' rebinding now obsolete